internal/application/item_service.go

```diff
@@ -11,16 +11,16 @@ import (
 	"github.com/teamcubation/go-items-challenge/pkg/log"
 )
 
-type itemService struct {
```

Comment:
Por que mudamos essa estrutura para maiúsculas (por que torná-la pública?) foi uma sugestão dos linters?

Reply:
Ok, pode reverter isso? Eu vejo isso como comum, já que a struct é usada apenas dentro do pacote e temos um método público New. O que pode estar acontecendo é que isso foi sugerido porque retornamos esse tipo no nosso método construtor. Talvez a melhor opção seja tornar a struct privada, mas na assinatura da função definir como tipo de retorno a interface (in.AuthService). Exemplo:
```
type authService struct {
	repo out.UserRepository
}

func NewAuthService(repo out.UserRepository) in.AuthService {
	return &authService{repo: repo}
}
```